application/helpers/user_info.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# require_relative '../validation'
+
+module UserInfo
+  def user_name
+    enter_name
+  end
+
+  def user_login
+    enter_login
+  end
+
+  def user_password
+    enter_password
+  end
+
+  def user_age
+    enter_age
+  end
```

Comment:
why do you need these methods?
all they do is just call another method

Reply:
removed methods  and made refactoring of their use.